cmd/fetchd/cmd/capricornmigrate.go

```diff
@@ -0,0 +1,489 @@
+				return fmt.Errorf("failed to delete wrong bridge contract: %w", err)
+			}
+
+			// update bridge and mobix contract codes
+			newBridgeContractPath, err := cmd.Flags().GetString(flagBridgeNewContractPath)
+			if err != nil {
+				return fmt.Errorf("failed to retrieve flag %q: %w", flagBridgeNewContractPath, err)
+			}
+			newMobixContractPath, err := cmd.Flags().GetString(flagMobixNewContractPath)
+			if err != nil {
+				return fmt.Errorf("failed to retrieve flag %q: %w", flagMobixNewContractPath, err)
+			}
+
+			appState, err = migrateWasmContracts(appState, cdc, newBridgeContractPath, newMobixContractPath)
+			if err != nil {
+				return fmt.Errorf("failed to migrate wasm contracts: %w", err)
+			}
+
+			appState, err = updateMobixContractState(appState, cdc)
+			if err != nil {
+				return fmt.Errorf("failed to migrate wasm contracts: %w", err)
+			}
+
+			slashingParamDoubleSignFractionStr, err := cmd.Flags().GetString(flagNewSlashFractionDoubleSign)
+			if err != nil {
+				return fmt.Errorf("failed to retrieve flag %q: %w", flagNewSlashFractionDoubleSign, err)
+			}
+
+			slashingParamDoubleSignFraction, err := sdk.NewDecFromStr(slashingParamDoubleSignFractionStr)
+			if err != nil {
+				return fmt.Errorf("invalid flag %q value: %w", flagNewSlashFractionDoubleSign, err)
+			}
+
+			appState, err = setNewSlashFractionDoubleSign(appState, cdc, slashingParamDoubleSignFraction)
+			if err != nil {
+				return fmt.Errorf("failed to set new slashing double sign fraction param: %w", err)
+			}
+
+			// Validate state (same as fetchd validate-genesis cmd)
+			if err := app.ModuleBasics.ValidateGenesis(cdc, clientCtx.TxConfig, appState); err != nil {
+				return fmt.Errorf("failed to validate state: %w", err)
+			}
+
+			// build and print the new genesis state
+			genDoc.AppState, err = json.Marshal(appState)
+			if err != nil {
+				return errors.Wrap(err, "failed to JSON marshal migrated genesis state")
+			}
+			bz, err := tmjson.Marshal(genDoc)
+			if err != nil {
+				return errors.Wrap(err, "failed to marshal genesis doc")
+			}
+			sortedBz, err := sdk.SortJSON(bz)
+			if err != nil {
+				return errors.Wrap(err, "failed to sort JSON genesis doc")
+			}
+
+			fmt.Println(string(sortedBz))
+
+			return nil
+		},
+	}
+
+	cmd.Flags().String(flagGenesisTime, "", "override genesis_time with this flag")
+	cmd.Flags().Int64(flagInitialHeight, 0, "override initial_height with this flag")
+	cmd.Flags().String(flags.FlagChainID, "fetchhub-3", "override chain_id with this flag")
+	cmd.Flags().Int64(flagConsensusBlockMaxBytes, 200_000, "override consensus.block.max_bytes with this flag")
+	cmd.Flags().Int64(flagConsensusBlockMaxGas, 2_000_000, "override consensus.block.max_gas with this flag")
+	cmd.Flags().String(flagFoundationAddress, "fetch1c2wlfqn6eqqknpwcr0na43m9k6hux94dp6fx4y", "fetch.ai foundation address")
+	cmd.Flags().String(flagFoundationTokensToBurn, "80616475991676767023462315afet", "fetch.ai foundation tokens to burn")
```

Comment:
nit: Maybe it's slightly more clear if we add the word 'amount' to the usage for `flagFoundationTokensToBurn`: `"amount of fetch.ai tokans to burn"`